tests/test_sap_success_factors/test_learner_data.py

```diff
@@ -302,3 +338,61 @@ def get_course_grade(course_id, username):
         assert learner_data[2].course_completed
         assert learner_data[2].completed_timestamp == self.NOW_TIMESTAMP
         assert learner_data[2].grade == grade
+
+    @ddt.data(
+        (True, True, 'audit', 1), (True, True, 'verified', 1),
+        (True, False, 'audit', 0), (True, False, 'verified', 1),
+        (False, True, 'audit', 0), (False, True, 'verified', 1),
+        (False, False, 'audit', 0), (False, False, 'verified', 1),
+    )
+    @ddt.unpack
+    @mock.patch('integrated_channels.integrated_channel.learner_data.EnrollmentApiClient')
+    @mock.patch('integrated_channels.integrated_channel.learner_data.GradesApiClient')
+    @mock.patch('integrated_channels.integrated_channel.learner_data.CourseApiClient')
+    def test_learner_data_audit_track_data_reporting_enabled(
+            self,
+            enable_audit_enrollment,
+            enable_reporting,
```

Comment:
@UmanShahzad Let's call this `enable_audit_data_reporting` to be as specific as possible.